packages/cli/src/project/attributes/env.ts

```diff
@@ -23,9 +23,9 @@ export class EnvAttribute implements IProjectAttribute {
 
   async csharp(targetDir: string) {
     const changeCase = await import('change-case');
-    const key = changeCase.pascalCase(this._key, {
-      delimiter: '.'
-    });
+    const key = this._key.split('.')
```

Comment:
(can you add a comment above on why this is needed)

Reply:
This ensures that  keys like "teams.clientId" are converted to "Teams.ClientId"
It follows the C# convention for environment variables.